openapi_tester/validators.py

```diff
@@ -25,6 +28,37 @@
 )
 
 
+def _create_format_validator(validator: Callable, wrap_as_validator: bool = False) -> Callable:
+    def wrapped(value: Any):
+        try:
+            result = validator(value)
+            return not wrap_as_validator or (wrap_as_validator and result)
+        except (ValueError, ValidationError):
+            return False
+
+    return wrapped
+
+
+_number_format_validator = _create_format_validator(
+    lambda x: isinstance(x, float) if x != 0 else isinstance(x, (int, float)), True
+)
+
+DJANGO_VALIDATOR_MAP = {
```

Comment:
Small detail: not sure about the naming; why did you call it *django* validator map?